src/sessionManager.ts

```diff
@@ -179,8 +210,7 @@ export default function SessionManager(
                     messageType: Types.MessageType.SessionEnd,
                 });
 
-                mpInstance._Store.sessionStartDate = null;
```

Comment:
I think line 182 was removed erroneously.

Reply:
I intentionally moved it to a new method `Store.nullifySession()` since it made sense to me to keep this grouped with the other nullify session logic. Is there a reason to not nullify the session start date when we nullify a session?